src/recensio/plone/behaviors/pages_of_presented_text.py

```diff
@@ -31,10 +31,27 @@ class IPagesOfPresentedText(model.Schema):
     # but I have to think more about it
     directives.mode(heading__page_number_of_presented_text_in_print="display")
 
+    fieldset(
+        "reviewed_text",
+        label=_("label_schema_reviewed_text", default="Reviewed Text"),
+        fields=[
+            "heading__page_number_of_presented_text_in_print",
+        ],
+    )
+
+    fieldset(
+        "presented_text",
```

Comment:
In general I would stick with the ID `reviewed_text`. As some behaviors can be used both for reviews and presentations this can refer either to the reviewed text or the presented text, but we don't know this here in the schema.

In the old code the method `finalize_recensio_schema` does some magic to make sure the fieldset labels are correct. Maybe we can ignore this for the moment as presentations are only a legacy type anyway.

Reply:
ok, right. makes sense to not follow the orignal code here. answers my question from above.